core/iwasm/include/wasm_c_api.h

```diff
@@ -191,10 +166,50 @@ typedef union MemAllocOption {
         void *user_data;
     } allocator;
 } MemAllocOption;
-#endif
+#endif /* MEM_ALLOC_OPTION_DEFINED */
+
+/* Runtime configration */
+struct wasm_config_t {
+    mem_alloc_type_t mem_alloc_type;
+    MemAllocOption mem_alloc_option;
+    bool linux_perf_support;
+    /*TODO: wasi args*/
+};
```

Comment:
Seems no need to expose wasm_config_t in this header file, is it better to define it in wasm_c_api.c?